feedback/locale/fi/LC_MESSAGES/django.po

```diff
@@ -464,8 +464,40 @@ msgid "Respond"
 msgstr "Vastaa"
 
 #: feedback/templates/manage/_response_message.html
-msgid "The student has seen this response."
-msgstr "Opiskelija on nähnyt tämän vastauksen."
+msgid "Toggle text preview"
+msgstr "Tekstin esikatselun vaihtaminen"
+
+#: feedback/templates/manage/_response_message.html
+msgid "Preview"
+msgstr "Esikatsele"
+
+#: feedback/templates/manage/_response_message.html
+msgid "Back to editing"
+msgstr "Palaa muokkausnäkymään"
```

Comment:
Would strings `Show preview` and `Hide preview` be better?

Reply:
"Show preview" sounds redundant, since "preview" would be enough. However, "Back to editing" could indeed be replaced with "Hide preview" or "Write" (as Zulip has). Opinions?

Reply:
"Hide preview" sounds the best to me.

Reply:
That's what I suspected you'd answer (and I also asked Gusse's opinion and he agreed), so I changed it to that. I decided to include a bit more info in the aria-label "Hide preview and return to editing", which probably is fine.